frontend/src/pages/_app.tsx

```diff
@@ -52,6 +55,49 @@ function App({ Component, pageProps }: AppPropsWithLayout): JSX.Element {
 
   return (
     <>
+      <Head>
```

Comment:
Move fallback SEO meta tags to `_app.tsx` from `_document.tsx`, so these meta tags are overwritable by page specific ones that share the same keys. Without doing this, the page will have duplicate meta tags even with the same keys